typegen.json

```diff
@@ -1,11 +1,17 @@
 {
   "outDir": "src/types",
-  "specVersions": "https://kusama.archive.subsquid.io/graphql",
+  "specVersions": "https://amplitude.archive.subsquid.io/graphql",
```

Comment:
Again: why were we referring to the kusama archive and not the amplitude?

Reply:
@RustNinja do you remember something about this? Is it just an oversight or was there a reason?